dev/prod_builders.json

```diff
@@ -264,6 +264,12 @@
          "task_name": "linux_fuchsia_precache",
          "flaky": false
       },
+      {
+        "name": "Linux gradle_desugar_classes_test",
+        "repo": "flutter",
+        "task_name": "linux_gradle_desugar_classes_test",
```

Comment:
@godofredoc I was going over https://github.com/flutter/flutter/blob/master/dev/README.md, is there anything else required to configure the prod/try builders?

Reply:
Yes if this is a new tasks then the builders should exist before adding these configurations. My recommendation is to land the code changes then create the configs in infra and the final step would be updating the prod|try_builders.json.

We just added some docs: https://g3doc.corp.google.com/company/teams/flutter/luci/builders.md?cl=head

Reply:
cool